### DISCUSSION TITLE
Project Collaborator Invitation

### DISCUSSION THREAD

**Discussion by atrafmustafayev1** [community/community#33184] [Other Feature Feedback, Questions, & Ideas]
Hello, I can not add collaborators to Github Project Board. Why? I invite people but they do not get neither emails nor notifications

**Comment by ck12330** [comment]
There is a problem with your Project Collaboration invitation so kindly solve this issue by check that the invitation person email ID or via which kind of service you are using to send invitation. I will suggest you to recheck the (,/./space) before sending and be confirmed that your are updated.

**Comment by ck12330** [comment]
This Code of Conduct is adapted from the [Contributor Covenant](https://www.contributor-covenant.org/), version 2.0, available at https://www.contributor-covenant.org/version/2/0/code_of_conduct.html.
